README.md

```diff
@@ -30,6 +31,14 @@ run `npm test`
 
 run `npm run lint:fix`
 
+## Load remote couchdb with design documents and default tag
+
+run `npm run migrate:couchdb-documents`: To save both design documents and default tags to couchdb
```

Comment:
This script is named `migrate:all` in the package.json. Keep it `:all`. That is better than the longer `couchdb-documents`
```suggestion
run `npm run migrate:all`: To save both design documents and default tags to couchdb
```

Reply:
My bad i forgot to change that..

Reply:
Done